### ISSUE TITLE
Readable format

### ISSUE THREAD

**Issue by jholt1** [forsti0506/a11y-sitechecker#167]
Is there a way to convert the JSON to a readable format, such as HTML?

I had a quick try with https://github.com/lpelypenko/axe-html-reporter. However, it looks like the format of the JSON here differs from the AXE format for violations.

**Comment by digitala11y** [comment]
As a blind person who use screen reader i prefer a CSV format. can we add this to the feature list?

**Comment by forsti0506** [comment]
@digitala11y yes i can add this to the feature list, but I'm not sure how to display complex data in csv format? can you maybe convert the following json example to a csv to help me undestand it?

`json
{
"errors": [{ "location": "some html", "selector" : "html"}, { "location": "some html2", "selector" : "html2"}], 
"passed": [{ "location": "some html", "selector" : "html"}, { "location": "some html2", "selector" : "html2"}], 
}
`

**Comment by digitala11y** [comment]
Hello,
I tried using the app, as a screen reader user I hit a wall. I am new to using command line and node. After installing node modules, I was lost on how to initiate a scan.

This need bit of guidance, I thought to use my scan results to csv and share here.

This package here also generates JSON report and I use JSON to CSV converter. The accessibility violations are easy to navigate and easy to import to bug tracking tools too.
https://github.com/GovTechSG/purple-hats
